From: Michal Hocko <mhocko@suse.com>
Subject: kabi: x86/speculation/l1tf: Increase l1tf memory limit for Nehalem+
Patch-mainline: no, suse specific
References: bnc#1105536

hide new x86_cache_bits from the kabi checker. Nobody should be touching
this part of the structure.

Signed-off-by: Michal Hocko <mhocko@suse.com>

---
 arch/x86/include/asm/processor.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/arch/x86/include/asm/processor.h
+++ b/arch/x86/include/asm/processor.h
@@ -110,8 +110,10 @@ struct cpuinfo_x86 {
 	/* Index into per_cpu list: */
 	u16			cpu_index;
 	u32			microcode;
+#ifndef __GENKSYMS__
 	/* Address space bits used by the cache internally */
 	u8			x86_cache_bits;
+#endif
 } __attribute__((__aligned__(SMP_CACHE_BYTES)));
 
 #define X86_VENDOR_INTEL	0
